docs/application/tizen-studio/rt-ide/rt-overview.md

```diff
@@ -1,12 +1,23 @@
-# RT IDE
+# Tizen RT
 
-The Tizen Studio for RT is an IDE that provides a lightweight RTOS (real-time operating system)-based application development environment that helps you develop, build, flash, and debug applications in Ubuntu.
+Tizen has been commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The goal of TizenRT is to extend the Tizen platform device coverage to these kind of low-end devices.
```

Comment:
Minor changes, please rephrase to:

Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of Tizen RT is to extend the Tizen platform device coverage to these kind of low-end devices.